src/main/java/br/com/caelum/carangobom/config/security/TokenService.java

```diff
@@ -33,6 +37,14 @@ public String generateToken(Authentication authentication) {
             .compact();
     }
 
+    public static String retrieveTokenFromHeaderValue(String authorizationHeaderValue) {
+        if(authorizationHeaderValue == null || authorizationHeaderValue.isEmpty() || !authorizationHeaderValue.startsWith("Bearer ")) {
```

Comment:
suggestion: quebra de linha.
```suggestion
        if(authorizationHeaderValue == null || 
        authorizationHeaderValue.isEmpty() || 
        !authorizationHeaderValue.startsWith("Bearer ")) {
```